### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by ScienceMaster7**
What I did:

- I created a fonts.css file and use custom variables.
- I made some color changes.
- I styled the forms.

### CODE REVIEW

File: src/pages/Menu.css [hunk 1]
```diff
@@ -15,6 +15,6 @@
   margin: 20px;
   padding: 15px 60px;
```
Comment (hellodebs) [inline src/pages/Menu.css:16]: Is there a reason why you use px here but rem everywhere else for padding and margin?

File: src/styles/colors.css [hunk 2]
```diff
@@ -1,5 +1,5 @@
 :root {
-  --main-bg-color: #36a7fd;
+  --main-bg-color: beige;
   --primary-color: #fd7d36;
   --primary-text-color: #ffffff;
   --sencodary-text-color: #000000;
```
Comment (hellodebs) [inline src/styles/colors.css:5]: I like structure. Well done. :)

**Review by hellodebs — COMMENTED** [review]
good job!

File: src/styles/colors.css (hunk 2 above)
Comment (ScienceMaster7) [reply src/styles/colors.css:5]: thanks :)

File: src/pages/Menu.css (hunk 1 above)
Comment (ScienceMaster7) [reply src/pages/Menu.css:16]: not realy, must have overlooked it.

File: src/pages/CreateHabit.css [hunk 3]
```diff
@@ -11,13 +11,18 @@
 }
 
 .CreateHabit__input {
+  font-size: var(--normal-text);
```
Comment (mikemangas) [inline src/pages/CreateHabit.css:14]: im not sure if we should use bem for the colors too? maybe you can check that out?

File: src/pages/Home.css [hunk 4]
```diff
@@ -20,12 +21,12 @@
   border-style: initial;
   padding: 2rem;
   margin: 1rem;
-  font-size: 1.5rem;
+  font-size: var(--normal-text);
   cursor: pointer;
 }
 
 .Home__text--no-habbits {
```
Comment (mikemangas) [inline src/pages/Home.css:28]: nice bem example here :)

**Pull request merged**